server.js

```diff
@@ -59,7 +62,7 @@ app.get('/project', function(req,res) {
 		if(err) {
```

Comment:
Does this endpoint do anything yet? If so, you should be doing more than just sending a status.